src/components/organisms/Settings/Project/Plugin/hooks.ts

```diff
@@ -1,28 +1,37 @@
 import { useLocalState } from "@reearth/state";
+import { useInstallablePluginsQuery, useInstalledPluginsQuery } from "@reearth/gql";
+import { useMemo } from "react";
```

Comment:
All react imports should be at the very top

Reply:
Thank you!